package.json

```diff
@@ -45,6 +46,7 @@
     ]
   },
   "jest": {
+    "testTimeout": 15000,
```

Comment:
@samj1912 I assume you are timing out on localhost when running the tests?

Reply:
Removed.